### PR TITLE
Scrum 36 header

### PR DESCRIPTION
**Pull request by Xer0ce**

### CODE REVIEW

**Review by soualmatthias — DISMISSED** [review]

File: .github/workflows/a-rea-mirror.yml
Comment (albe2) [inline .github/workflows/a-rea-mirror.yml:1]: dont delete this file

File: front/web/public/discord.png
Comment (albe2) [inline front/web/public/discord.png:1]: prefere an https link then an asset

File: front/web/public/spotify.png
Comment (albe2) [inline front/web/public/spotify.png:1]: same here

**Review by albe2 — CHANGES_REQUESTED** [review]

**Review by albe2 — APPROVED** [review]
sgtm

**Pull request merged**